src/i18n/messages/fr.messages.json

```diff
@@ -93,10 +93,10 @@
       "addToCart": "AJOUTER AU PANIER"
     },
     "spellcheck": {
-      "message": "Aucun résultat trouvé pour <span class=\"x-text x-text--accent x-text--bold\">'{query}'</span>. Nous vous montrons les résultats pour"
+      "message": "Aucun résultat trouvé pour <span class=\"x-font-bold  x-text-auxiliary-50\">'{query}'</span>. Nous vous montrons les résultats pour"
```

Comment:
should be a title3?

Reply:
I'm treating the spellcheck component as something like a message component and using the styles for that one. I'm asking design anyway for confirmation

![image](https://user-images.githubusercontent.com/7124620/209829991-fbb618ea-6987-4bb4-aeea-25c87f4b3c52.png)

Reply:
I'm treating it as a message component, the same as the no results one, and it's `text-1`

![image](https://user-images.githubusercontent.com/7124620/209830864-5a10369f-3d80-4453-b8e9-e20ed2a365b4.png)